### PR TITLE
added Proof of Completed Service forms, modify form list/bank

### PR DESCRIPTION
**Pull request by oliviaclyde**
I've added the Proof of Service form to the static form bank, modified the form list and form bank

### CODE REVIEW

**Comment by joeldenning** [comment]
This PR's build is failing because our code doesn't yet work with Node 13, which recently came out. To fix it, please merge master into your branch.

```sh
git remote -v # find the name for both the central repo and your fork's repo
git checkout master
git pull <name_of_central_remote> master
git checkout olivia-add_forms_to_form_list_and_bank
git merge master
git push -u <name_of_fork_remote>
```

File: frontend/app/forms/bank/form-bank.component.js [hunk 1]
```diff
@@ -170,14 +170,22 @@ export default class FormBank extends React.Component {
...
+        />
         <Route
           path={`${this.props.match.url}/petition-for-expungement-from-bci`}
           render={props => (
             <GovernmentForm
               name="Petition for Expungement (Special Certificate from BCI)"
               WebForm={PetitionForExpungementFromBci_Web}
               PdfForm={PetitionForExpungementFromBci_Pdf}
-            />
+         />
```
Comment (joeldenning) [inline frontend/app/forms/bank/form-bank.component.js]: I'm pretty sure that when you merge master in to fix the build, that you'll get a new build failure for the formatting of this code. We use a tool called [Prettier](https://prettier.io/) that auto-formats our code so we don't have to worry about doing it ourselves. We also use [husky](https://github.com/typicode/husky) and [pretty-quick](https://github.com/azz/pretty-quick) which set up prettier automatically for all people contributing to the codebase. The following code in our package.json shows this:

https://github.com/JustUtahCoders/utahexpungements.org/blob/6386186e1f0650eed09249e89313551f38e7bccd/package.json#L17-L21

What I'm trying to show here is that you seem to have accidentally bypassed all of this when you set up your local version of the code. Try the following to fix:

```sh
rm -rf node_modules # this doesn't work on windows, but just delete the entire node_modules directory
yarn install
yarn prettier 'frontend/**/*' --write

# now add and commit
git commit -a -m "Fixing code formatting"
```

^ Note that you normally don't have to do these steps, but that your local version of the code somehow bypassed all of this. The normal way to bypass is to do `git commit --no-verify` intentionally, which I doubt you did. So it must have been a problem with the installation that prevented the git hook from working.

**Comment by joeldenning** [comment]
The pull request is now showing 25 files changed, which is incorrect. Somehow your branch got messed up in such a way that github is showing files that already exist as new files.

I will look into this further and post here when I discover what's happening.

**Comment by oliviaclyde** [comment]
Dang it. I was screen-sharing with Dan and we thought we got it to work. I'm happy to delete the branch and just start from scratch.

**Comment by joeldenning** [comment]
Looks like I was right -- this appears to just be a bug in Github's Pull Request diffing because when it was merged it was only the 9 files that changed, not the 25 files. See https://github.com/JustUtahCoders/utahexpungements.org/commit/943aa8520e1aa405a49c4203ad2c47937db18363

**Pull request merged**